stock_picking_force_assign/data/ir_config_parameter.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo noupdate="1">
+    <record model="ir.config_parameter" id="parameter_force_assign_allow_partial">
+        <field name="key">stock_picking_force_assign.allow_partial</field>
```

Comment:
@StefanRijnhart Would it be more flexible to configure it on stock.picking.type level ?

Reply:
Yes, there can be a lot of ways to implement the configuration of this behaviour: on the picking type, or on the company or depending on the picking policy of a related sale order etc. Our requirement, in a complex multicompany, multiwarehouse setup is just a change in the global behaviour, I would want to avoid a configuration by picking type because we have a lot of picking types, and there is already a constant state of confusion with regards to the existing settings on this level up to the point where the behaviour of Odoo seems unpredicatable and unreliable to its warehouse users.